external-crates/move/crates/move-compiler/tests/sui_mode/linter/lint_all_syntax_missing.exp

```diff
@@ -0,0 +1,6 @@
+warning[W10007]: issue with attribute value
+  ┌─ tests/sui_mode/linter/lint_all_syntax_missing.move:2:14
+  │
+2 │ #[lint_allow(x)]
+  │              ^ Unknown warning filter 'lint(x)'
```

Comment:
This error message is rather confusing. Can you special-case it?

Reply:
What would you like to see?

Reply:
`Unknown warning filter 'lint_allow'` would be ideal. I'd just be very confused seeing this as a user.